tests/functional/tests/bundles/base.yaml

```diff
@@ -0,0 +1,13 @@
+applications:
+  ubuntu:
+    charm: ch:ubuntu
+    num_units: 1
+
+  software-inventory-exporter:
+    charm: ch:software-inventory-exporter
+    channel: latest/edge # this should be removed after the charm is on stable
```

Comment:
I think we will always test this with latest/edge here.

Reply:
I think that snap should have such functional tests, so any new changes should be tested with other stable things. But I agree that maybe we could have two bundles here, one with edge snap and one with stable.

Reply:
Currently we don't have charm options to set the snap channel, but I think it's a good idea adding it. Here we are just using the latest/edge of `software-inventory-exporter` charm until we have a release.

I guess this is a scope for another PR in the future